hackportal.config.ts

```diff
@@ -527,6 +571,8 @@ interface CheckboxQuestion extends RegistrationQuestion {
   options: Array<{
     title: string;
     value: string;
+    link: string;
+    linkText: string;
```

Comment:
Thoughts on these 2 fields optional? Meaning the interface will look like the following: 
```ts
options: Array<{
  title: string;
  value: string;
  link?: string;
  linkText?: string;
}>
```
That way, we won't have to add `link` and `linkText` to every other textbox questions.

Reply:
Sounds good! Thank you, I was unsure of how to do that.